xDripG5/Messages/GlucoseRxMessage.swift

```diff
@@ -8,39 +8,10 @@
 
 import Foundation
 
-
-public struct GlucoseRxMessage: TransmitterRxMessage {
-    static let opcode: UInt8 = 0x31
-    let glucoseIsDisplayOnly: Bool = false  // TODO
-    let status: UInt8
-    let sequence: UInt32
-    public let timestamp: UInt32
-    public let glucose: UInt16
-    public let state: UInt8
-    public let trend: Int8
-
-    init?(data: NSData) {
-        if data.length >= 14 {
-            if data[0] == self.dynamicType.opcode {
-                status = data[1]
-                sequence = data[2...5]
-                timestamp = data[6...9]
-                glucose = data[10...11] & 0xfff
-                state = data[12]
-                trend = data[13]
-            } else {
-                return nil
-            }
-        } else {
-            return nil
-        }
+public protocol GlucoseRxMessage: TransmitterRxMessage {
```

Comment:
I'm not sure this is necessary to fulfill any DI needs in your client tests, since this is just a DTO. Instead, change the failable initializer to a `convenience` init and reimplement the default struct initializer to accept straightforward values.